docs/quick-start.md

```diff
@@ -42,9 +42,16 @@ helm install --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defa
 
 The following charts should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
 
-* `kubewarden-crds`, which will register the `ClusterAdmissionPolicy` and `PolicyServer` Custom Resource Definitions
+* `kubewarden-crds`, which will register the `ClusterAdmissionPolicy`,
+  `AdmissionPolicy` and `PolicyServer` Custom Resource Definitions.  As well as
+  the `PolicyReport` Custom Resource Definitions used by the audit scanner
 
-* `kubewarden-controller`, which will install the Kubewarden controller
+* `kubewarden-controller`, which will install the Kubewarden controller and the
+  audit scanner
+:::note
+If you want to disable the audit scanner component. Please check out the audit
+scanner installation [docs page](../howtos/audit-scanner).
+:::
 
 * `kubewarden-defaults`, which will create a `PolicyServer` resource named `default`. It can also installs a set of
```

Comment:
```suggestion
* `kubewarden-defaults`, which will create a `PolicyServer` resource named `default`. It can also install a set of
```

Reply:
Also, a sentence a but further on which I don't appear to be able to add a change to. I'm not sure how best to do that. But, anyway. Line 70 change to ''all policies bound to it'

Reply:
for that line 70 change, you need to submit a new pull request, as it is out of bounds of the changes in this one. I will resolve this review conversation, we can keep this topic of this pull request.

Reply:
Ok, thanks.